src/pages/components/table.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Table
+description: >
+  Tables organize and display related content from a data set. This helps users
```

Comment:
_Edited for clarity:_

A Table organizes data into columns and rows. Tables make it easier for users to compare and analyze information.